[0m11:29:41.905504 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025032289220>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002503238A180>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000250323891F0>]}


============================== 11:29:41.914504 | 618336f5-567a-4c37-99a3-88e75df33b0a ==============================
[0m11:29:41.914504 [info ] [MainThread]: Running with dbt=1.8.3
[0m11:29:41.915504 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Val\\.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt init demo_dbt', 'send_anonymous_usage_stats': 'True'}
[0m11:29:41.922504 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': WindowsPath('C:/Users/Val/.dbt')}
[0m11:29:41.923504 [info ] [MainThread]: Creating dbt configuration folder at 
[0m11:29:41.924504 [debug] [MainThread]: Starter project path: C:\Users\Val\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\include\starter_project
[0m11:29:41.931504 [info ] [MainThread]: 
Your new dbt project "demo_dbt" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m11:29:41.932504 [info ] [MainThread]: Setting up your profile.
[0m11:31:06.894135 [info ] [MainThread]: Profile demo_dbt written to C:\Users\Val\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m11:31:06.895135 [debug] [MainThread]: Command `dbt init` succeeded at 11:31:06.895135 after 85.08 seconds
[0m11:31:06.896135 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025032A5B2C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025033E99C10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002503706ADB0>]}
[0m11:31:06.896135 [debug] [MainThread]: Flushing usage events
[0m12:41:31.966554 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000015847F47DA0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000158483049E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000158484388C0>]}


============================== 12:41:31.969554 | 15a6d8ae-e0c0-4335-906f-c72147ba88f2 ==============================
[0m12:41:31.969554 [info ] [MainThread]: Running with dbt=1.8.3
[0m12:41:31.969554 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': 'C:\\Users\\Val\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m12:41:31.970554 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\Val\dev\data_engineering\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m12:41:31.970554 [debug] [MainThread]: Command `dbt run` failed at 12:41:31.970554 after 0.08 seconds
[0m12:41:31.971554 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000158478972C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000015847F8CB30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000158486068D0>]}
[0m12:41:31.971554 [debug] [MainThread]: Flushing usage events
[0m12:46:11.400913 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000225AEB93740>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000225B180A060>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000225B0EE7FE0>]}


============================== 12:46:11.403913 | 51c16a0a-899e-42e8-a923-084a0c0fe889 ==============================
[0m12:46:11.403913 [info ] [MainThread]: Running with dbt=1.8.3
[0m12:46:11.404912 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Val\\.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'debug': 'False', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init test_dbt', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m12:46:11.412913 [debug] [MainThread]: Starter project path: C:\Users\Val\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\include\starter_project
[0m12:46:11.422913 [info ] [MainThread]: 
Your new dbt project "test_dbt" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m12:46:11.423913 [info ] [MainThread]: Setting up your profile.
[0m12:46:48.862916 [info ] [MainThread]: Profile test_dbt written to C:\Users\Val\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m12:46:48.863917 [debug] [MainThread]: Command `dbt init` succeeded at 12:46:48.863917 after 37.52 seconds
[0m12:46:48.863917 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000225AEB93740>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000225FD53FB00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000225B425A330>]}
[0m12:46:48.863917 [debug] [MainThread]: Flushing usage events
[0m13:56:50.935914 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002710E986ED0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002710D2E8DA0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002710E985880>]}


============================== 13:56:50.937909 | fdd0d111-e3d3-4a62-af72-f513253554b4 ==============================
[0m13:56:50.937909 [info ] [MainThread]: Running with dbt=1.8.3
[0m13:56:50.938910 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Val\\.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt run', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m13:56:50.938910 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\Val\dev\data_engineering\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m13:56:50.939912 [debug] [MainThread]: Command `dbt run` failed at 13:56:50.939912 after 0.06 seconds
[0m13:56:50.939912 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002710E618F80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002710D2E8DA0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002710D4A8D70>]}
[0m13:56:50.939912 [debug] [MainThread]: Flushing usage events
